src/uu/echo/src/echo.rs

```diff
@@ -123,7 +120,6 @@ pub fn uumain(args: impl uucore::Args) -> i32 {
         .setting(clap::AppSettings::TrailingVarArg)
         .setting(clap::AppSettings::AllowLeadingHyphen)
         .version(crate_version!())
-        .usage(USAGE)
```

Comment:
why did you remove it?
it was useful
we could use get_usage like the rest of the program,

Reply:
ping?

Reply:
:)
I also value consistency. I changed it back. 

However for me it still makes no sense, as you need to write a lot of extra code with no added value (in the ls example the function get_usage() is only used at exactly one spot, where the built-in pattern of clap could be leveraged, and instead you need to maintain a string and a fn. But maybe I miss something here.